feature/feature.go

```diff
@@ -40,6 +40,7 @@ const (
 	// alpha: v0.3
 	// beta: v0.4
 	// GA: v1.10
+	// Deprecated: ClusterResourceSet feature is now GA and the corresponding feature flag will be removed in 1.12 release.
```

Comment:
```suggestion
        //
	// Deprecated: ClusterResourceSet feature is now GA and the corresponding feature flag will be removed in 1.12 release.
```
I think staticcheck only picks that up with a leading empty comment line (which is also why the linter didn't complain)